### DISCUSSION TITLE
Release Plan for v1.2

### DISCUSSION THREAD

**Discussion by HajimeKawahara** [HajimeKawahara/exojax#298] [General]
See [here](https://github.com/HajimeKawahara/exojax/milestone/3) as the current plan for v1.2.

- maybe after the release of `radis 0.14`, because the default molecular API in v1.2 should use radis.api #256 #289 
- need to rewrite tutorials and check all of them works c.f. [How?](https://semaphoreci.com/blog/test-jupyter-notebooks-with-pytest-and-nbmake).

**Comment by erwanp** [comment]
@anandxkumar  will help with the release of radis 0.14 but I'm currently stuck ! :)

**Reply by HajimeKawahara** [reply]
OK, thanks. No need to rush!

**Comment by HajimeKawahara** [comment]
We are in the process of releasing [v1.2](https://github.com/HajimeKawahara/exojax/milestone/3). Thanks a lot for releasing `radis 0.14` @anandxkumar !

**Reply by anandxkumar** [reply]
Awesome :rocket:
